packages/lit-element/src/lib/lit-element.ts

```diff
@@ -126,58 +125,42 @@ export class LitElement extends UpdatingElement {
    * using the [[`css`]] tag function or via constructible stylesheets.
    */
   static styles?: CSSResultOrNative | CSSResultArray;
```

Comment:
One problem with the current typings is that is messes up subclassing.

```ts
class A extends LitElement {
  static styles = css`...`;
}

class B extends A {
  static styles = [A.styles, css`...`]; // Error CSSResult[] not assignable to CSSResult
}
```

We could fix this by making `css`'s return type be the same as `static styles`.

Reply:
It's a trade off. If we do that users of `CSSResult` can't really use its API without casting. I guess that's mostly internal and tests.

Instead, we could require an array? It's a breaking change but very mechanical to fix. Wdyt?

Reply:
I don't like requiring an array. We should try to drive down as much boilerplate as possible, symbol-by-symbol.

Reply:
Let's handle this in a separate PR.